dlt-database/entity/0001-init_db/Community.ts

```diff
@@ -46,19 +37,15 @@ export class Community {
   @JoinColumn({ name: 'auf_account_id' })
   aufAccount?: Account
 
-  @CreateDateColumn({ name: 'created_at', type: 'datetime', default: () => 'CURRENT_TIMESTAMP(3)' })
+  @Column({ name: 'created_at', type: 'datetime', default: () => 'CURRENT_TIMESTAMP(3)' })
```

Comment:
If the `createdAt` column will only contain the timestamp of the write access than the annotation `@CreateDateColumn` will be prefered.
But keep in mind:
the `@CreateDateColumn` and/or `@UpdateDateColumn` will not treat the ms-dimension properly